src/main/java/bot/services/BotService.java

```diff
@@ -76,9 +76,9 @@ public void shutdownBot(){
         this.jda.shutdown();
     }
 
-    public Command getCommandWithName(String name){
-        return null;
-    }
+   // public Command getCommandWithName(String name){
```

Comment:
> Task :compileJava
[49](https://github.com/Andreichenko/Java-discord/runs/6397371571?check_suite_focus=true#step:5:49)
/home/runner/work/Java-discord/Java-discord/src/main/java/bot/controllers/SendCommandController.java:35: error: cannot find symbol
[50](https://github.com/Andreichenko/Java-discord/runs/6397371571?check_suite_focus=true#step:5:50)
        Command command = botService.getCommandFromName(commandName);
[51](https://github.com/Andreichenko/Java-discord/runs/6397371571?check_suite_focus=true#step:5:51)
                                    ^
[52](https://github.com/Andreichenko/Java-discord/runs/6397371571?check_suite_focus=true#step:5:52)
  symbol:   method getCommandFromName(String)
[53](https://github.com/Andreichenko/Java-discord/runs/6397371571?check_suite_focus=true#step:5:53)
  location: variable botService of type BotService
[54](https://github.com/Andreichenko/Java-discord/runs/6397371571?check_suite_focus=true#step:5:54)
/home/runner/work/Java-discord/Java-discord/src/main/java/bot/utils/EmbedUtils.java:45: error: cannot find symbol
[55](https://github.com/Andreichenko/Java-discord/runs/6397371571?check_suite_focus=true#step:5:55)
                    audioTrack.getInfo().uri, TimeLineStamp.timeString(audioTrack.getDuration() / 1000)));
[56](https://github.com/Andreichenko/Java-discord/runs/6397371571?check_suite_focus=true#step:5:56)
                                              ^
[57](https://github.com/Andreichenko/Java-discord/runs/6397371571?check_suite_focus=true#step:5:57)
  symbol:   variable TimeLineStamp